app/views/steps/caution/result/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<% title 'Check when your criminal record expires' %>
+
+<div class="govuk-width-container">
+  <%= step_header %>
+
+  <main class="govuk-main-wrapper">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <div class="govuk-panel govuk-panel--confirmation">
+          <h1 class="govuk-panel__title">
+            Your caution expired on <%=  @expiry_date.strftime('%d %B %Y') %>
+          </h1>
+          <div class="govuk-panel__body">
+            You do not need to tell anyone about the caution unless an <a class="govuk-link" href="https://www.gov.uk/exoffenders-and-employment?#when-employers-still-need-to-know-about-your-conviction">exception applies</a>.
+          </div>
+        </div>
+
+        <h2 class="govuk-heading-m">How this date has been worked out</h2>
+
+        <p class="govuk-body">The expiry date of your caution is based on:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>the information you’ve provided</li>
+          <li>the current law on when cautions and convictions expire</li>
+        </ul>
+
+        <p class="govuk-body">
+          This table shows the information you provided, which was used to work out the expiry date. If you entered any incorrect information, the expiry date will be incorrect.
+        </p>
+
+        <dl class="govuk-summary-list">
+          <%= render partial: 'steps/caution/result/shared/row',
+                     collection: @presenter, as: :row %>
```

Comment:
Perhaps better to be explicit and call here `@presenter.summary` instead of doing it in the controller.

That way we can even, if you want to, refactor a bit the presenter to have 2 methods, one for the `#summary` and another one for the `#expiry_date`.